packages/peregrine/lib/talons/OrderHistoryPage/useOrderHistoryPage.js

```diff
@@ -4,13 +4,11 @@ import { useQuery } from '@apollo/client';
 
 import { useAppContext } from '../../context/app';
 import { useUserContext } from '../../context/user';
-import { useTypePolicies } from '../../hooks/useTypePolicies';
+import DEFAULT_OPERATIONS from './orderHistoryPage.gql';
 
-export const useOrderHistoryPage = props => {
-    const { queries, types } = props;
-    const { getCustomerOrdersQuery } = queries;
-
-    useTypePolicies(types);
+export const useOrderHistoryPage = (props = {}) => {
+    const { operations = DEFAULT_OPERATIONS } = props;
```

Comment:
@revanth0212 - There was overwhelming support for your proposed pattern of merging operations; do you want to propose a solution in this PR? I don't recall what we said needed to happen with classify and generalizing the shallow merge utility.

Reply:
Honestly, I don't remember either if we wanted to create a utility or just do the shallow merge in every PR, but I can add a utility and surrounding code in another branch carved out of this. If the team is ok with it, we can merge it into this PR. How does that sound?